src/mame/misc/micom_mahjong.cpp

```diff
@@ -0,0 +1,237 @@
+// license:BSD-3-Clause
+// copyright-holders:hap
+/******************************************************************************
+
+マイコン麻雀 - Micom Mahjong, mail-order Mahjong console.
+
+Two models are known, a grey one and a white one, hardware is presumed to be
+the same. It's not known who developed/manufactured it. Probably not the
+mail company, but they can be considered the publisher.
```

Comment:
Might be better to say “…the mail order house…” rather than “…the mail company…” as the latter could be interpreted as referring to the postal service.  Nippon Mail Service (日本メールサービス) is just a mail order house.  The postal service was previously Yūsei Jigyōchō (郵政事業庁), then Japan Post (日本郵政公社 – Nippon Yūsei Kōsha), and now Japan Post Holdings (日本郵政株式会社 – Nippon Yūsei K.K.).